setup.py

```diff
@@ -25,7 +25,6 @@
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
-        'License :: OSI Approved :: License (ISCL)',
```

Comment:
AGPL is like GPL but hosting on a web server, eg with a cloud linter requires AGPL compliance.